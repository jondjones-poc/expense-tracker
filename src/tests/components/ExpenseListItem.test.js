import React from 'react';
import { shallow } from 'enzyme';
import ExpenseListItem from '../../components/expenseListItem';
import expensesTestData from '../Fixtures/expensesTestdata';

test('Should render expense list item correctly', () => {
    const wrapper = shallow(<ExpenseListItem {...expensesTestData[0]} />);
    expect(wrapper).toMatchSnapshot();
});
