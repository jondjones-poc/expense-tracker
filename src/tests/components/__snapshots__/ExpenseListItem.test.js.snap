// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render expense list item correctly 1`] = `
<li>
  <Link
    replace={false}
    to="/edit/1"
  >
    <h3>
      rent
    </h3>
  </Link>
  <p>
    $0.10
    -
    January 1st, 1970
  </p>
</li>
`;
